criticality_score/run.py

```diff
@@ -255,7 +255,11 @@ def recent_releases_count(self):
             days_since_creation = self.created_since * 30
             if not days_since_creation:
                 return 0
-            total_tags = self._repo.get_tags().totalCount
+            total_tags = 0
+            try:
+                total_tags = self._repo.get_tags().totalCount
+            except Exception:
+                logger.error(f'get_tags is failed: {self._repo.url}')
```

Comment:
Add a comment that this failure is usually due to large number of tags.
Also, i think we should default to 26, or whatever that constant for max release count is (use constant from that file). this just means we default to max threshold which is equivalent to having any value larger than it (since it caps it).

Reply:
Okay, added the comment: `# Very large number of tags, i.e. 5000+. Cap at 26.`
And returning `RECENT_RELEASES_THRESHOLD` as the default value